### PR TITLE
Payee 1159

### PR DESCRIPTION
**Pull request by vivekhmrc**
- [ ] Unit tests passing
- [ ] Coverage reached
- [ ] Acceptance tests passing
- [ ] Reviewed

### CODE REVIEW

**Review by sergi-hmrc — APPROVED** [review]

**Comment by sergi-hmrc** [comment]
you may want to add something like in the else statement: 
`if(rtiToggle.rtiEnabled && taxYear.year >= TaxYear().next.year) {
        logger.info(s"RTIAPI - Bad request error received: ${res.body}")
      }`

**Review by Jake-Raffe — APPROVED** [review]

**Review by brendanstraughan — APPROVED** [review]

File: app/uk/gov/hmrc/tai/connectors/RtiConnector.scala [hunk 1]
```diff
@@ -130,6 +130,7 @@ class RtiConnector @Inject()(
         }
       }
     } else {
+      logger.info(s"RTIAPI - call for CY+ year: $taxYear}")
```
Comment (sergi-hmrc) [inline app/uk/gov/hmrc/tai/connectors/RtiConnector.scala:133]: surround this logger with: 
`if(rtiToggle.rtiEnabled && taxYear.year >= TaxYear().next.year) {`

**Review by PerrettJ4 — APPROVED** [review]

File: app/uk/gov/hmrc/tai/connectors/RtiConnector.scala (hunk 1 above)
Comment (sergi-hmrc) [reply app/uk/gov/hmrc/tai/connectors/RtiConnector.scala:133]: otherwise you'll logg for CY+ for any year if rtiEnabled is disabled

Comment (sergi-hmrc) [reply app/uk/gov/hmrc/tai/connectors/RtiConnector.scala:133]: If you want to capture everything then I'd suggest changin the message to something more explicit like:
Not calling RTI API for year: year

Comment (sergi-hmrc) [reply app/uk/gov/hmrc/tai/connectors/RtiConnector.scala:133]: or Skip RTI API - call for year

Comment (vivekhmrc) [reply app/uk/gov/hmrc/tai/connectors/RtiConnector.scala]: Done

**Pull request merged**